scripts/dts/edtlib.py

```diff
@@ -890,6 +890,23 @@ def __repr__(self):
             "binding " + self.binding_path if self.binding_path
                 else "no binding")
 
+    def _init_status(self):
+        # Initializes Node._node_status.
+        #
+        # Node._node_status holds node enabled condition. This is dynamically
+        # evaluated by the node status property and If his parent status is
```

Comment:
s/his/its/

Pronouns for dead things are never he/she in English, though I know where it's coming from. :)

Reply:
Ops...text changed!